resources/language_server_node_cdx/launcher.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/bash
-
-exec npm start
+echo 'Starting server.js using node...' >launcher.log
+./link_logs.sh &>/dev/null </dev/null &
+exec ./nodejs/bin/node ./src/server.js --stdio --strict-code-completion
```

Comment:
would nodejs be found like that? or would it be better to push this invocation down into the "start" script of our lsp.zip?

Reply:
Executing node or npm (or trying to locate it via 'which') in the ssh console does not find it. So I guess this is a contract that node is located at ./nodejs/bin/node i.e. starting it via npm start will also not improve things.
The link_logs.sh was a try out but did never work (=>remove). Guess we have no permission to write something in the ~/logs directory. Nevermind. Would be interesting if the LSP infrastructure does support telemetry/event request. Log file is currently reset upon restart of LS.